pkg/controller/linstorsatelliteset/linstorsatelliteset_controller.go

```diff
@@ -46,11 +46,14 @@ import (
 	"github.com/piraeusdatastore/piraeus-operator/pkg/apis/piraeus/shared"
 	piraeusv1 "github.com/piraeusdatastore/piraeus-operator/pkg/apis/piraeus/v1"
 	mdutil "github.com/piraeusdatastore/piraeus-operator/pkg/k8s/metadata/util"
+	"github.com/piraeusdatastore/piraeus-operator/pkg/k8s/monitoring"
 	"github.com/piraeusdatastore/piraeus-operator/pkg/k8s/reconcileutil"
 	kubeSpec "github.com/piraeusdatastore/piraeus-operator/pkg/k8s/spec"
 	lc "github.com/piraeusdatastore/piraeus-operator/pkg/linstor/client"
 )
 
+const DefaultMonitoringImage = "quay.io/piraeusdatastore/drbd-reactor:v0.3.0"
```

Comment:
Our other default images are in a `const.go` file. Shall we do that here too for consistency?